mlir/lib/Dialect/MIOpen/Transforms/AffineTransforms.cpp

```diff
@@ -57,12 +57,46 @@ AffineMap AffineTransforms::buildIndexAffineMap(miopen::TransformOp op) {
         assert(srcDimAttr.size() == destDimAttr.size());
 
         auto parameters = dimLayoutAttr.get("parameters").dyn_cast<ArrayAttr>();
-        auto leftPad = parameters.getValue()[0].dyn_cast<IntegerAttr>().getInt();
+        // auto leftPad =
+        // parameters.getValue()[0].dyn_cast<IntegerAttr>().getInt();
         for (unsigned j = 0; j < srcDimAttr.size(); ++j) {
+          // [0, 2, 3, 1] leftpadH = 0 rightPadH = 2 leftpadW = 3 rightPadW = 1
```

Comment:
this paragraph better be re-formatted to say it's an example, otherwise it's hard to understand what `[0, 2, 3, 1]` means.